7.Lifecycle.md

```diff
@@ -0,0 +1,29 @@
+**Jam3 Lessons - React**
+
+# Lifecycle
+
+> Each component has several "lifecycle methods" that you can override to run code at particular times in the process.
+Methods prefixed with will are called right before something happens, and methods prefixed with did are called right after something happens.
```

Comment:
Quotes are all copied from the React docs, if you think we should describe on our own, I could rewrite it. I just thought that they wrote it really accurate and very easy to understand, I can't find anything to change.